### PR TITLE
Incorporated feedback

### PR DESCRIPTION
**Pull request by bukunmig**
fix: changed from port forward to exposed

refactor: updated to point to consul k8s 1.1.2 . Need to clarify if 1.15.3 will be available.

docs: Updated README for reccs to export tokens and deploy HashiCups. 

Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,25 @@
...
+
+export AWS_ACCESS_KEY_ID=... 
+export AWS_SECRET_ACCESS_KEY=...
+export AWS_SESSION_TOKEN=...
+
+```
+
+After deploying the mesh, deploy HashiCups demo app. 
+
+Demo can be found here: https://github.com/hashicorp-education/learn-consul-get-started-kubernetes/tree/main/self-managed/eks/hashicups
```
Comment (gautambaghel) [inline README.md:28]: I think the idea here is to show folks how they can deploy hashicups here, so creating a folder here and copying yaml files would be better, please make sure the deployment actually works. thanks!

File: main.tf [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Copyright (c) HashiCorp, Inc.
 # SPDX-License-Identifier: Apache-2.0
 
-# TODO: fix this
+# DONE: fixed to version 1.1.2
```
Comment (gautambaghel) [inline main.tf:4]: Let's delete this line

**Review by gautambaghel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bukunmig) [reply README.md:28]: Suggested edits have been made.

File: main.tf (hunk 2 above)
Comment (bukunmig) [reply main.tf:4]: Deleted

**Comment by bukunmig** [comment]
Suggested edits have been made.

**Review by gautambaghel — APPROVED** [review]

**Comment by gautambaghel** [comment]
LGTM!

**Review by gautambaghel — APPROVED** [review]
LGTM!

**Pull request merged**